src/SourceBuild/content/eng/tools/init-build.proj

```diff
@@ -113,13 +113,14 @@
     </Touch>
   </Target>
 
-  <Target Name="BuildSdkArchiveDiff"
+  <Target Name="BuildUnifiedBuildValidationTests"
           Condition="'$(ShortStack)' != 'true' and '$(PortableBuild)' == 'true' and '$(PgoInstrument)' != 'true'" >
-    <MSBuild Projects="tasks\Microsoft.DotNet.SourceBuild.Tasks.SdkArchiveDiff\Microsoft.DotNet.SourceBuild.Tasks.SdkArchiveDiff.csproj"
+    <MSBuild Projects="$(UnifiedBuildValidationTestsProject)"
              Targets="Restore"
-             Properties="MSBuildRestoreSessionId=$([System.Guid]::NewGuid())" />
-    <MSBuild Projects="tasks\Microsoft.DotNet.SourceBuild.Tasks.SdkArchiveDiff\Microsoft.DotNet.SourceBuild.Tasks.SdkArchiveDiff.csproj"
-             Targets="Build" />
+             Properties="MSBuildRestoreSessionId=$([System.Guid]::NewGuid());ExcludeFromDotNetBuild=false" />
+    <MSBuild Projects="$(UnifiedBuildValidationTestsProject)"
```

Comment:
`UnifiedBuildValidationTestsProject` property isn't available here. I would just hardcode the full path here as we for the other projects.